emailUtil.ts

```diff
@@ -10,15 +10,13 @@ function alertBody(
   tag: TrackTag,
   camera: String
 ): string[] {
-  const dateTime = moment(recording.recordingDateTime)
-    .tz(config.timeZone)
```

Comment:
is setting the time zone not needed anymore?

Reply:
doh thanks